app/helpers/application_helper/toolbar/servicetemplate_center.rb

```diff
@@ -36,7 +36,6 @@ class ApplicationHelper::Toolbar::ServicetemplateCenter < ApplicationHelper::Too
         button(
           :catalogitem_copy,
           'pficon pficon-edit fa-lg',
-          N_('This Item cannot be copied'),
```

Comment:
`button(id, icon, title, text, keys = {})` .. we do need 2 labels, 

something like 

```
t = N_('Copy Selected Item'),
t,
```
?

Reply:
good catch, fixed.